src/client/tile.h

```diff
@@ -154,6 +154,7 @@ video::IImage * Align2Npot2(video::IImage * image, video::IVideoDriver* driver);
 #endif
 
 enum MaterialType{
+	TILE_MATERIAL_OPAQUE,
```

Comment:
I don't think it really matters here but you're right, that is usually good practice.